modules/troubleshooting-network-observability-controller-manager-pod-out-of-memory.adoc

```diff
@@ -6,26 +6,41 @@
 [id="controller-manager-pod-runs-out-of-memory_{context}"]
 = Network Observability controller manager pod runs out of memory
 
-You can increase memory limits for the Network Observability operator by patching the Cluster Service Version (CSV), where Network Observability controller manager pod runs out of memory.
+You can increase memory limits for the Network Observability operator by editing the `spec.config.resources.limits.memory` configuration in the `Subscription` object.
 
 .Procedure
 
-. Run the following command to patch the CSV:
+. In the web console, navigate to *Operators* -> *Installed Operators*
+. Click *Network Observability* then select *Subscription*.
+. From the *Actions* menu, choose *Edit Subscription*.
```

Comment:
"Choose" indicates many choices. Is this the case? Maybe stick with "Click" to avoid any confusion?